recipes/pygubu/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pygubu" %}
+{% set version = "0.35.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygubu-{{ version }}.tar.gz
+  sha256: 7b6dcb543da74d5e8a5329d20855ed0dccb561c3431f93d8b1a7902530fae4cf
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - importlib-resources  # [py<39]
+
+test:
```

Comment:
This should have an `import` test of the package:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-imports